tests/sentry/grouping/grouping_inputs/cocoa_dispatch_client_callout.json

```diff
@@ -0,0 +1,114 @@
+{
+  "platform": "cocoa",
+  "message": "Foo",
+  "threads": {
+    "values": [
+      {
+        "stacktrace": {
+          "frames": [
+            {
+              "function": "unicorn",
+              "package": "/private/var/containers/Bundle/Application/9B2E1C43-8839-4525-9586-FC58F0534940/Foobar.app/Foobar",
+              "in_app": false,
+              "data": {
+                "orig_in_app": 1
+              }
+            },
+            {
+              "function": "UIApplicationMain",
+              "package": "/System/Library/PrivateFrameworks/UIKitCore.framework/UIKitCore",
+              "in_app": false,
+              "data": {}
+            },
+            {
+              "function": "-[UIApplication _run]",
+              "package": "/System/Library/PrivateFrameworks/UIKitCore.framework/UIKitCore",
+              "in_app": false,
+              "data": {}
+            },
+            {
+              "function": "_dispatch_main_queue_drain",
+              "package": "/usr/lib/system/libdispatch.dylib",
+              "in_app": false,
+              "data": {}
+            },
+            {
+              "function": "_dispatch_client_callout",
```

Comment:
You will see in the snapshot that we will not try applying `category:threadbase -group v-group`.